src/edges_estimate/foregrounds.py

```diff
@@ -220,6 +220,22 @@ def model(self, **p):
         return p["amp"] * np.sin(2 * np.pi * self.freqs / p["lambda"] + p["phase"])
 
 
+@attr.s
+class Sinusoid_cossin(Foreground):
```

Comment:
Class names should be CamelCase without underscores. I suggest `DampedOscillations` ?